webpack/assets/javascripts/react_app/components/notifications/NotificationDrawerToggle.js

```diff
@@ -23,10 +24,18 @@ class NotificationDrawerToggle extends Component {
   onChange(actionType) {
     switch (actionType) {
       case ACTIONS.RECEIVED_NOTIFICATIONS: {
-        this.setState({
-          count: NotificationsStore.getNotifications().length,
-          isLoaded: true
-        });
+        const notifications = NotificationsStore.getNotifications();
+
+        const newState = _.reduce(notifications, (curr, next) => {
```

Comment:
I think that If we should need it in the future, we will implement. Currently, if its not needed, it should be removed